### PR TITLE
Added missing spec files for API800/1000/1200 for all network resources

### PR DESCRIPTION
**Pull request by yuvirani**
### Description
Added missing spec files for API800/1000/1200 for all network resources

### Check List
- [ ] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
- [ ] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [ ] New endpoints supported are updated in the endpoints-support.md file.
- [ ] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: examples/shared_samples/ethernet_network.rb [hunk 1]
```diff
@@ -32,6 +40,15 @@
 # api_version = 500 & variant = Synergy to OneviewSDK::API500::Synergy::EthernetNetwork
 # api_version = 600 & variant = C7000 to OneviewSDK::API600::C7000::EthernetNetwork
 # api_version = 600 & variant = Synergy to OneviewSDK::API600::Synergy::EthernetNetwork
+# api_version = 800 & variant = Synergy to OneviewSDK::API800::Synergy::EthernetNetwork
+# api_version = 800 & variant = C7000 to OneviewSDK::API800::C7000::EthernetNetwork
+# api_version = 1000 & variant = C7000 to OneviewSDK::API1000::C7000::EthernetNetwork
+# api_version = 1000 & variant = Synergy to OneviewSDK::API1000::Synergy::EthernetNetwork
+# api_version = 1200 & variant = C7000 to OneviewSDK::API1200::C7000::EthernetNetwork
+# api_version = 1200 & variant = C7000 to OneviewSDK::API1200::C7000::EthernetNetwork
```
Comment (chebroluharika) [inline examples/shared_samples/ethernet_network.rb:48]: change variant to Synergy

File: examples/shared_samples/fc_network.rb [hunk 2]
```diff
@@ -32,6 +40,15 @@
 # api_version = 500 & variant = Synergy to OneviewSDK::API500::C7000::FCNetwork
 # api_version = 600 & variant = C7000 to OneviewSDK::API600::C7000::FCNetwork
 # api_version = 600 & variant = Synergy to OneviewSDK::API600::C7000::FCNetwork
+# api_version = 800 & variant = C7000 to OneviewSDK::API800::C7000::FCNetwork
+# api_version = 800 & variant = Synergy to OneviewSDK::API800::C7000::FCNetwork
```
Comment (chebroluharika) [inline examples/shared_samples/fc_network.rb:44]: Resource created should be for Synergy

File: examples/shared_samples/fc_network.rb [hunk 3]
```diff
@@ -32,6 +40,15 @@
...
+# api_version = 1000 & variant = C7000 to OneviewSDK::API1000::C7000::FCNetwork
+# api_version = 1000 & variant = Synergy to OneviewSDK::API1000::C7000::FCNetwork
```
Comment (chebroluharika) [inline examples/shared_samples/fc_network.rb:46]: same as above

File: examples/shared_samples/fc_network.rb [hunk 4]
```diff
@@ -32,6 +40,15 @@
...
+# api_version = 1200 & variant = C7000 to OneviewSDK::API1200::C7000::FCNetwork
+# api_version = 1200 & variant = Synergy to OneviewSDK::API1200::C7000::FCNetwork
```
Comment (chebroluharika) [inline examples/shared_samples/fc_network.rb:48]: same as above

File: examples/shared_samples/fc_network.rb [hunk 5]
```diff
@@ -32,6 +40,15 @@
...
+# api_version = 1600 & variant = C7000 to OneviewSDK::API1600::C7000::FCNetwork
+# api_version = 1600 & variant = Synergy to OneviewSDK::API1600::C7000::FCNetwork
```
Comment (chebroluharika) [inline examples/shared_samples/fc_network.rb:50]: same as above

File: spec/unit/resource/api1600/synergy/uplink_set_spec.rb [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+RSpec.describe OneviewSDK::API1600::Synergy::UplinkSet do
+  include_context 'shared context'
+
+  it 'inherits from OneviewSDK::API1200::Synergy::UplinkSet' do
+    expect(described_class).to be < OneviewSDK::API1200::Synergy::UplinkSet
+  end
+
+  describe '#initialize' do
+    it 'should be initialize the instance with default values' do
+      item = described_class.new(@client_1000)
```
Comment (chebroluharika) [inline spec/unit/resource/api1600/synergy/uplink_set_spec.rb:12]: This should be @client_1600

**Comment by yuvirani** [comment]
> change copyright years in example files. Add copyrights in all spec files.

Added copy rights in all spec files,

File: examples/shared_samples/ethernet_network.rb (hunk 1 above)
Comment (chebroluharika) [inline examples/shared_samples/ethernet_network.rb:48]: I am still not able to see variant Synergy. Please change it. And please have a look in all other files and for other apiversions.

File: examples/shared_samples/uplink_set.rb [hunk 7]
```diff
@@ -17,7 +17,7 @@
 # NOTE 3: To add an ethernet network, the interconnect must support ethernet network.
```
Comment (chebroluharika) [inline examples/shared_samples/uplink_set.rb:17]: Change copyright year in this file.

**Pull request closed without merging**